distributions/otelcol-contrib/.goreleaser.yaml

```diff
@@ -55,7 +55,7 @@ nfpms:
         type: config|noreplace
       - src: config.yaml
         dst: /etc/otelcol-contrib/config.yaml
-        type: config
+        type: config|noreplace
```

Comment:
Please do the same to the Core distribution

Reply:
I've also patched the core distribution in the same way :)